app/models/move.rb

```diff
@@ -51,6 +51,8 @@ class Move < VersionedModel
   validates :reference, presence: true
 
   validates :status, inclusion: { in: statuses }
+  # we need to avoid creating a move with the same person/date/from/to if there is already an 'active' one.
+  validates :date, uniqueness: { scope: %i[status person_id from_location_id to_location_id] }, on: :create
```

Comment:
this index will prevent multiple cancellations: I think they should be permitted